docs/DeveloperGuide.md

```diff
@@ -223,157 +235,179 @@ The activity diagram shows the general sequence of steps when a user interacts w
+  exists in the AddressBook.
+  - **Pros:** Ensures data integrity by preventing appointments from being linked to non-existent persons.
+  - **Cons:** Requires additional validation logic during the edit process.
+- **Alternative 2:** Execution of the `edit appt` command does not check if the new `personId` exists in the AddressBook.
+  - **Pros:** Simplifies the edit process by removing the need for additional validation.
+  - **Cons:** Increases the risk of data inconsistency, as appointments may be linked to non-existent persons.
 
 <br>
 
-#### Find Person/Appointment feature
-
-**Aspect: How to show find person/appointment based on different criteria**
+#### Find Person / Appointment feature
 
+**Aspect: How to implement the `find person` and `find appt` commands that allow finding by multiple criteria**
 - **Alternative 1 (Current choice)**: Create one find command that supports filtering by multiple criteria (name, date) using prefixes.
-  - Pros: Fast and easy to find by date and name
-  - Cons: Confusing syntax from user's perspective
-
+  - Pros: Fast and easy to find by date and name.
+  - Cons: Confusing syntax from user's perspective.
 - **Alternative 2**: Create different find commands, find by date, find by name etc.
-  - Pros: Much easy in terms of user experience
-  - Cons: More repeated code for each command
+  - Pros: Much easy in terms of user experience.
+  - Cons: More repeated code for each command.
 
 <br>
 
-**Aspect: How to combine multiple prefixes when finding results**
-
+**Aspect: How to combine multiple prefixes when executing the `find` commands**
 - **Alternative 1 (Current choice)**: Prefixes should be combined using an AND condition.
-  - Pros: Ensures more specific search results, as all conditions must be met
-  - Cons: May be too restrictive
-
+  - **Pros**: Ensures more specific search results, as all conditions must be met.
+  - **Cons**: May be too restrictive.
 - **Alternative 2**: Prefixes should be combined using an OR condition
-  - Pros: Allows for more flexible and broader search results, as any one of the conditions can yield matches.
-  - Cons: May return too many results
+  - **Pros**: Allows for more flexible and broader search results, as any one of the conditions can yield matches.
+  - **Cons**: May return too many results.
 
 <br>
 
-**Aspect: Whether to implement case sensitivity in matching**
-
+**Aspect: Whether to implement case sensitivity in matching for search terms**
 - **Alternative 1 (Current choice)**: Implement case-insensitive matching for search terms.
-    - **Pros**: Enhances user experience by allowing searches to ignore case differences
-    - **Cons**: Slightly more processing required to normalize case during search
-
+    - **Pros**: Enhances user experience by allowing searches to ignore case differences.
+    - **Cons**: Slightly more processing required to normalize case .during search
```

Comment:
There is a slight errata here with the fullstop in the middle of the sentence. Please fix!